core/src/main/java/io/undertow/server/SecureCookieCommitListener.java

```diff
@@ -0,0 +1,22 @@
+package io.undertow.server;
+
+import io.undertow.server.handlers.Cookie;
+
+import java.util.Map;
+
+/**
+ * Sets the <pre>secure</pre> attribute on all response cookies.
+ */
+public enum SecureCookieCommitListener implements ResponseCommitListener {
```

Comment:
I don't know if this should be a public class, I don't think it really has much of an application outside the SecureCookieHandler

Reply:
Agreed, but the alternative is making `HttpServerExchange.getResponseCookiesInternal` public, currently it's package private.

Reply:
Alternatively it may be cleaner to implement `HttpServerExchange.forEachCookie(Consumer<Cookie> consumer)`

Reply:
In that case I think this is fine. Would you be able to file a JIRA for this and include it in the commit message? In general every commit should be attached to a JIRA, and otherwise I need to ammend your commits locally to update the message.

Reply:
Filed and updated, thanks!